components/docs-chef-io/content/automate/servicenow_reference.md

```diff
@@ -376,3 +377,74 @@ The ServiceNow app deletes these records of the corresponding closed incidents,
+
+You can assign the `x_chef_event.admin` role to a user other than a System Administrator to allow another user to manage the application properties and logs.
+
+The System Administrator authorization includes access to the tasks in the **Admin** role.
+
+The **Admin** role grants user access to the following:
+
+- Chef Events menu
+- Client Infra Client Run Alerts menu item
+- Chef InSpec scans Alerts menu item
+- Client Infra Client Runs
+- Chef InSpec scans
+- Properties menu item
+- Support menu item
+- Logs menu item
+
+### Role `x_chef_event.chef_inspec_scans_user`
+
+The `x_chef_event.chef_inspec_scans_user` role is suitable for those users who require application access without administration rights. The role grants a user access to the:
+
+- Chef Events menu
+- Chef Infra scans Alerts menu item
+- Chef InSpec scans menu item
+- Support menu item
+
+{{< note >}}
+
+Client run and Chef InSpec scan records are linked to events, and is appropriate to assign this role to users who manage events in ServiceNow.
+
+{{< /note >}}
+
+### Role x_chef_event.api
+
+The `x_chef_event.api` role is suitable for users responsible for integrating the Chef Automate data into ServiceNow. We recommend creating a new user specifically for this role. The Chef Automate Event Creation App requires the API role to set up communication with Chef Automate.
+
+### Event Creation App Properties
+
+`x_chef_event.client_run_message_key`
+: Used to associate a Chef Infra client run record with an event record. Possible values: `cookbook` and `node`. Default: `cookbook`.
+
+   Setting the value to `cookbook` creates an event for the failed cookbook. This associate all failing Chef Infra client runs with the corresponding event. `cookbook` is the default value because the number of nodes exceeds the number of cookbooks in any system. The message key description of the event provides information about the failure.
+
+   ![Chef Events Client Runs page] data not available
+
+   The **Chef Infra Client Run Alerts** tab of the alerts displays the associated client run events. Setting the value to `node` creates an event for each failed node. All failing Chef Infra client runs for a node associated with the corresponding event. The message key description of the event provides information about the run failure for a node.
+
+   ![Chef Events Client Run Alerts page](/images/automate/sn-event-clientrun-alerts.png)
+
+`x_chef_event.inspec_scan_message_key`
+
+: Used to associate a Chef InSpec scan record with an event record. Possible values: `profile` and `node`. Default: `profile`.
+
+   Setting the value to `profile` creates an event for the failed chef InSpec compliance profile with a message value. This associate all failing Chef InSpec compliance scans with the corresponding alerts. `profile` is the default value because the number of nodes exceeds the number of cookbooks in any system. The message key description of the event provides information about the failure.
```

Comment:
Fix the markdown, clarify "This associate all failing Chef InSpec compliance scans with the corresponding alerts." and I don't think we need "`profile` is the default value because the number of nodes exceeds the number of cookbooks in any system."

Reply:
Sure, will check with Eng. team and remove it.

Reply:
@vivekshankar1 : as discussed, would you please let us know your thoughts on this?